homeassistant/components/lcn/services.yaml

```
@@ -520,8 +520,8 @@ lock_keys:
         text:
     table:
       name: Table
-      description: "Table with keys to lock (must be A for interval)."
-      example: "a5"
+      description: "Table with keys to lock (optional, must be A for interval)."
```

There is no `required` attribute specified in the service description of this field, thus optional is assumed.

```suggestion
      description: "Table with keys to lock (must be A for interval)."
```